README.md

```diff
@@ -10,29 +10,30 @@ Current Version: **17.1.20171115**
 
 You can launch the image using the docker command line,
 
-```bash
-mkdir -p /opt/mysql-egroupware/data
-docker run -d \
---name mysql-egroupware \
--e MYSQL_ROOT_PASSWORD=MySuperSecretSqlPassword \
--e MYSQL_DATABASE=egroupware \
--e MYSQL_USER=egroupware \
--e MYSQL_PASSWORD=AnotherSecretPassword \
--v /opt/mysql-egroupware/data:/var/lib/mysql \
-mysql
+```
+#!/usr/bin/env bash
```

Comment:
This is supposed to be a snippet to show the commands involved to launch the docker containers necessary.
With a shebang it looks like a ready to run script. Please remove this line.